district_data_utils.py

```diff
@@ -49,7 +49,7 @@ def create_dp_donorrecord(district_record, school_year):
     """Create a DP donorrecord from the given district record (without creating any studentrecords)."""
     if not school_year:
         raise ValueError("school_year param required")
-    main_l_name = district_record['Parent 1 Last Name']
+    main_l_name = district_record['Parent1 Last Name']
```

Comment:
@kdawgsf, we could create an alias for these fields and an input table that maps the incoming data header to our aliases so that we don't have to tweak multiple places in the code with label changes. But it may be overkill for how often we actually do this...